tests/mocha/variant_browser.js

```diff
@@ -92,6 +98,30 @@ function variantSearchBySpeciesandChrLocation(driver){
         });
     });
 }
+
+function positionFilterBoxValidation(driver){
+    driver.findElement(By.id("selectFilter-trigger-picker")).click();
+    driver.findElement(By.xpath("//li[text()='Chromosomal Location']")).click();
+    driver.findElement(By.id("speciesFilter-trigger-picker")).click();
+    driver.findElement(By.xpath("//li[text()='Mosquito / AaegL3']")).click();
+    driver.findElement(By.name("region")).clear();
+    driver.findElement(By.name("region")).sendKeys('supercont1.18:165624-165624');
```

Comment:
tests are updated.
1,13:12233-12234 this should open  alert box with "Please enter a valid region" msg.